helper/chunkify_test.js

```diff
@@ -0,0 +1,40 @@
+
+function chunkify(message, maxChars){
+    let lines = message.split(/\n/);
+    let chunks = [];
+    let numChars = 0;
+    for (let i = 0; i < lines.length; i++){
+        numChars = lines[i].length;
+        if (numChars < maxChars){
+            chunks.push(lines[i]);
+        } else {
+            let loc = lines[i].lastIndexOf(" ");

```

Comment:
use a more informative variable name, and const wherever possible

Reply:
Resolved in latest fix